maven-plugin-tools-generators/src/main/java/org/apache/maven/tools/plugin/generator/PluginDescriptorGenerator.java

```diff
@@ -45,6 +42,8 @@
 import org.codehaus.plexus.util.xml.PrettyPrintXMLWriter;
 import org.codehaus.plexus.util.xml.XMLWriter;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
Where is this used in the diff?

Reply:
old item, import reorder ...
used in lines: 101, 103

Reply:
checkstyle takes care about unused imports